packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -47,7 +47,7 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.2.1
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.14.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 5.0.0
```

Comment:
This needs a bump to <7 (or dropping all together as we did in Katello)

Reply:
Dropped the upper bound